cmd/ocm-backplane/testJob/testJob.go

```diff
@@ -0,0 +1,37 @@
+package testJob
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewTestJobCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "testjob",
+		Aliases:      []string{"testJob", "testjobs", "tj"},
+		Short:        "Represents a backplane testJob.",
+		SilenceUsage: true,
```

Comment:
Let's hide this command 
```suggestion
		SilenceUsage: true,
		Hidden:       true,
```

Reply:
done